graphdatascience/graph/graph_proc_runner.py

```diff
@@ -106,7 +106,8 @@ def list(self, G: Optional[Graph] = None) -> DataFrame:
 
     @client_only_endpoint("gds.graph")
     def get(self, graph_name: str) -> Graph:
-        if not self.exists(graph_name)["exists"]:
+        result = self._query_runner.run_query(f"CALL gds.graph.list('{graph_name}') YIELD graphName")
+        if len(result["graphName"]) == 0:
```

Comment:
We don't have a test for this using an `admin` user from what I can see. I think we should add one.

Reply:
I don't think so. That would be like testing that pagerank generates the correct result in a client test in my opinion. I.e. I think it's a server side concern that should be (and is) tested there